NonSucking.Framework.Serialization/NoosonGenerator.cs

```diff
@@ -412,6 +412,13 @@ private static string TypeNameToSummaryName(string typeName)
             }
         }
 
+        private static bool BaseHasNoosonAttribute(INamedTypeSymbol typeSymbol)
```

Comment:
```suggestion
        private static bool BaseHasNoosonAttribute(INamedTypeSymbol typeSymbol)
            => typeSymbol is not null 
                && (typeSymbol.GetAttribute(AttributeTemplates.GenSerializationAttribute) is not null 
                    || BaseHasNoosonAttribute(typeSymbol.BaseType))
```

Reply:
This can be made into one line as use the is null / is not null pattern matching style